src/picohttpparser/picohttpparser.c

```diff
@@ -282,14 +282,14 @@ static const char *parse_headers(const char *buf, const char *buf_end, struct ph
             /* parsing name, but do not discard SP before colon, see
              * http://www.mozilla.org/security/announce/2006/mfsa2006-33.html */
             headers[*num_headers].name = buf;
-            static const char ranges1[] __attribute__((aligned(16))) = "\x00 "  /* control chars and up to SP */
-                                                                       "\"\""   /* 0x22 */
-                                                                       "()"     /* 0x28,0x29 */
-                                                                       ",,"     /* 0x2c */
-                                                                       "//"     /* 0x2f */
-                                                                       ":@"     /* 0x3a-0x40 */
-                                                                       "[]"     /* 0x5b-0x5d */
-                                                                       "{\377"; /* 0x7b-0xff */
+            static const char ALIGNED(16) ranges1[] = "\x00 "  /* control chars and up to SP */
```

Comment:
How portable `ALIGNED` is between compilers, is this part of some standard and which C compilers implement it.